snappy_wrappers/wrappers/gcnv/post_germline_calls/wrapper.py

```diff
@@ -25,14 +25,25 @@
     fi
 done
 
+# Get contig name style
+# Note: the sample_index in '{snakemake.input.ploidy}' and '{snakemake.input.calls[0]}' are NOT guaranteed to be the same
+# -> Sample_0 should always exist and the contig names (which we care for) are the same
+tail -n +2 $(dirname {snakemake.input.ploidy})/ploidy-calls/SAMPLE_0/contig_ploidy.tsv | grep "chr" > /dev/null && true
```

Comment:
See the other comment about being more specific with the grep pattern, might be fine, as long as the input really has no stray "chr" that is not a contig prefix for whatever reason.

Reply:
Also made this regex more specific